ui/ui-config/__tests__/integration/happyPath.ts

```diff
@@ -113,21 +109,4 @@ describe("integration", () => {
     },
     helper.mediumTime
   );
-
-  test("if all the setup steps are complete, visiting / redirects to model", async () => {
-    await helper.factories.model({ name: "User", id: "user", type: "profile" });
-
-    const setupSteps = await SetupStep.findAll();
-    for (const step of setupSteps) {
-      await step.update({ complete: true });
-    }
-
-    await browser.get(`${url}/`);
-
-    await helper.sleep(1000);
-
-    const currentUrl = await browser.getCurrentUrl();
-    expect(currentUrl).toMatch(/\/model\/user\/overview/);
-    await browser.get(currentUrl);
-  });
```

Comment:
This test is OK to remove because we are testing this logic in the component.  These E2E tests shouldn't test conditional logic, but are here for making sure stuff boots and loads in general